src/main/java/me/ryanhamshire/GriefPrevention/EntityEventHandler.java

```diff
@@ -89,14 +92,10 @@ public EntityEventHandler(DataStore dataStore, GriefPrevention plugin)
     public void onEntityFormBlock(EntityBlockFormEvent event)
     {
         Entity entity = event.getEntity();
-        if (entity.getType() == EntityType.PLAYER)
```

Comment:
is there a reason to use instanceof instead of comparing type? I thought type was a bit faster than instanceof?

Reply:
It's negligible and provides us a guaranteed-safe inlined cast - previously we cast like 3 lines down. Hasn't been a problem yet, but I worry that one day a NPC plugin is going to make us sad.

Reply:
Oh, I didn't know that was a cast, I was a bit curious about that syntax. Is that a new java feature or have I just not seen anyone else ever do that?

Reply:
Java 17 feature, pattern variable if there's a match. https://docs.oracle.com/en/java/javase/17/language/pattern-matching-instanceof-operator.html

I used a couple of them in the damage handler work I did too I believe, don't remember for sure though

Reply:
neato, thanks!